lua/rest-nvim/request/init.lua

```diff
@@ -41,55 +33,34 @@ end
 -- @param bufnr Buffer number, a.k.a id
 -- @param start_line Line where body starts
 -- @param stop_line Line where body stops
--- @param has_json True if content-type is set to json
-local function get_body(bufnr, start_line, stop_line, has_json)
+-- @return table { external = bool; filename_tpl or body_tpl; }
+local function get_body(bufnr, start_line, stop_line)
   -- first check if the body should be imported from an external file
   local importfile = get_importfile_name(bufnr, start_line, stop_line)
-  local lines
+  local lines -- an array of strings
   if importfile ~= nil then
-    if not utils.file_exists(importfile) then
-      error("import file " .. importfile .. " not found")
-    end
-    lines = utils.read_file(importfile)
+    return { external = true, filename_tpl = importfile }
   else
-    lines = vim.api.nvim_buf_get_lines(bufnr, start_line - 1, stop_line, false)
+    lines = vim.api.nvim_buf_get_lines(bufnr, start_line, stop_line, false)
```

Comment:
I am a bit afraid removing the -1 might create bugs down the line but it was incorrect, i.e, if you look at the main branch today the request in tests/basic_get.http
```
GET https://reqres.in/api/users?page=5
```
loads with `GET https://reqres.in/api/users?page=5` as body xD

This means we could use golden (also called "snapshot") tests to make sure this is not the case. This PR helps making rest.nvim more "api-like" so I think it's a move in the good direction overall.